### DISCUSSION TITLE
How to find and use TIFF library?

### DISCUSSION THREAD

**Discussion by ghost** [JulNadeauCA/BSDBuild#17] [General]
https://cmake.org/cmake/help/latest/module/FindTIFF.html

**Comment by JulNadeauCA** [comment]
A `tiff` module was added in https://github.com/JulNadeauCA/BSDBuild/commit/f23e49abccd3893c1a215f0f5f39a9c6557beb75. It tests for both `libtiff` and `libtiffxx`. It has been tested with tiff 4.4.0.
Thanks for the suggestion!
